<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="inpu_csv_label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Input CSV</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="strategy_module_location">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_csv_location">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="strategy_module_label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Strategy Module</string>
    </property>
   </widget>
   <widget class="QPushButton" name="browseCSV">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="browseModule">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="execute_button">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>360</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Execute</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>331</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>Parameters</string>
    </property>
    <widget class="QDateEdit" name="start_date">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>110</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="threshold_label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Threshold</string>
     </property>
    </widget>
    <widget class="QLabel" name="returnsInCalculation_label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Returns in calculation</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="returnsInCalculation">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>60</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="threshold">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>30</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="decimals">
      <number>4</number>
     </property>
    </widget>
    <widget class="QLabel" name="start_date_label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Start Date</string>
     </property>
    </widget>
    <widget class="QLabel" name="end_date_label">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>90</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>End Date</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="end_date">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>110</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="date_valid">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>140</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="execution_status">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>350</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="inputcsv_valid">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>inpu_csv_label</zorder>
   <zorder>strategy_module_location</zorder>
   <zorder>input_csv_location</zorder>
   <zorder>strategy_module_label</zorder>
   <zorder>browseCSV</zorder>
   <zorder>browseModule</zorder>
   <zorder>execute_button</zorder>
   <zorder>execution_status</zorder>
   <zorder>inputcsv_valid</zorder>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
